scripts/common-functions.js

```diff
@@ -323,6 +329,12 @@ async function commonFunctionMigrateToV3() {
   });
 }
 
+/** @param {Node} element */
+function isAnchor(element) {
+  if (!element instanceof HTMLElement) return false;
+  return element.tagName && element.tagName.toLowerCase() === 'a';
```

Comment:
```suggestion
  if (!(element instanceof HTMLElement)) return false;
  return element.tagName && element.tagName.toLowerCase() === 'a';
```
As written, this checks if `!element` is an instance of `HTMLElement`, which I don't think will ever be true. Adding parentheses should fix it

Although this could probably be handled more elegantly as a one-liner by taking advantage of optional chaining.

Reply:
Nice catch!